pkg/field/defaults.go

```
@@ -56,6 +56,23 @@
 		WithExportTarget(ExportTargetOps))
 	skipFullSync            = BoolField("skip-full-sync", WithDescription("This must be set to skip a full sync"), WithPersistent(true), WithExportTarget(ExportTargetNone))
 	targetedSyncResourceIDs = StringSliceField("sync-resources", WithDescription("The resource IDs to sync"), WithPersistent(true), WithExportTarget(ExportTargetNone))
+	diffSyncsField = BoolField(
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)